### PR TITLE
fork and build 0.1.1

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] forked from conda-forge
* [x] checked requirements from [upstream](https://github.com/pymc-labs/pymc-marketing/blob/main/pyproject.toml)
* [x] checked license

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,25 +10,29 @@ source:
...
+    - setuptools >=61.0.0
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - arviz >=0.13.0
     - matplotlib-base >=3.5.1
-    - numpy >=1.17
+    - numpy {{ numpy }}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - numpy >=1.17
```
This pinning is preferable if we do not have numpy C-API in `host`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,25 +10,29 @@ source:
...
     - pandas
-    - pymc >=5.3.0
+    - pymc >=5.5.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - pymc >=5.3.0
```
Should be >=5.3.0 and the upstream comment also says that the minimum version needed https://github.com/pymc-labs/pymc-marketing/blob/ee587a8992d1bcffb616cb9638b340e80ca2edb6/pyproject.toml#L20

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,25 +10,29 @@ source:
...
     - scikit-learn >=1.1.1
     - seaborn >=0.12.2
     - xarray
+    - xarray-einstats >=0.5.1
+    - pymc-experimental >=0.0.8
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `pymc-experemental` is from the **main** branch made on **Jun 27** https://github.com/pymc-labs/pymc-marketing/blob/0855e0139c88719d2c3e16f59b5655a1a0df4115/pyproject.toml#L25 and not from **v0.1.1** https://github.com/pymc-labs/pymc-marketing/blob/0.1.1/pyproject.toml that has been released on **May 3** https://github.com/pymc-labs/pymc-marketing/releases/tag/0.1.1

Comment (boldorider4) [reply recipe/meta.yaml]: Oh you're right, thanks for catching that!

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**